src/webview2/mod.rs

```diff
@@ -1178,10 +1178,12 @@ impl InnerWebView {
   }
 
   pub fn bounds(&self) -> Result<Rect> {
-    let mut bounds = Rect::default();
+    let dpi = unsafe { util::hwnd_dpi(self.hwnd) };
+    let scale_factor = util::dpi_to_scale_factor(dpi);
```

Comment:
we don't need these, since we can just use Physical position without converting to logical